src/instructlab/model/download.py

```diff
@@ -304,3 +304,11 @@ def download_models(
     data = list_models([Path(model_dir)], False)
     data_as_lists = [list(item) for item in data]
     print_table(["Model Name", "Last Modified", "Size"], data_as_lists)
+
+
+def read_hf_token_from_file():
+    token_file = os.path.expanduser("~/.huggingface/token")
```

Comment:
This is a different path than in the above text

Reply:
sorry, use to test and forget to correct it